daemon/graphdriver/devmapper/README.md

```diff
@@ -216,3 +216,39 @@ Here is the list of supported options:
     Example use:
 
     ``docker -d --storage-opt dm.blkdiscard=false``
+
+ *  `dm.override_udev_sync_check`
+
+    Overrides the `udev` synchronization checks between devicemapper and udev.
+    `udev` is the device manager for the Linux kernel.
+
+    To view the `udev` sync support of a Docker daemon that is using the
+    devicemapper driver, run:
+
+        $ docker info
+	[...]
+	 Udev Sync Supported: true
+	[...]
+
+    When `udev` sync is supported, then devicemapper and udev can coordinate
+    the activation and deactivation of devices for containers.
+
+    When `udev` sync is `false`, a race conditions occurs between
+    the`devicemapper` and `udev` during create and cleanup. The race conditions
+    result in errors and failures. (For information on these failures, see
+    [docker#4036](https://github.com/docker/docker/issues/4036))
+
+    To allow the `docker` daemon to start, regardless of `udev` sync not being
+    supported, set `dm.override_udev_sync_check` to true:
+
+        $ docker -d --storage-opt dm.override_udev_sync_check=true
+
+    When this value is `true`, the  `devicemapper` continues and simply warns
+    you the errors are happening.
+
+    **Note**: The ideal is to pursue a `docker` daemon and environment that
```

Comment:
No it isn't. That is my fault. @vbatts I hate to pepper ya with this formatting. If you have time to update. I've fixed the gist:

https://gist.github.com/moxiegirl/ade0634692f7423bf474